hetio/neo4j.py

```diff
@@ -281,6 +281,171 @@ def construct_dwpc_query(metarels, property='name', join_hint='midpoint', index_
 
     return query
 
+def construct_pdp_query(metarels, dwpc=None, property='name', join_hint='midpoint', index_hint=False, unique_nodes=True):
+    """
+    Create a cypher query for computing the path degree product for a type of path.
```

Comment:
Really minor, but I think `cypher` should be capitalized (`Cypher`)

Reply:
You're right, I'll go ahead and update it